src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -330,6 +331,13 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
             if (ov::is_type<ov::op::internal::GatherCompressed>(node)) {
                 // It is necessary to avoid precision conversion for constant node(compressed weights)
                 ov::enable_keep_const_precision(node->get_input_node_shared_ptr(0));
+
+                if (ov::intel_cpu::one_of(node->get_input_node_shared_ptr(0)->get_element_type(),
```

Comment:
Lets add comment here with the description. Smt like: "Prioritize LPT pipeline to handle dequantization part for quantized models as it more optimal in general case"